src/Sylius/Bundle/ApiBundle/Resources/config/api_platform/resources/admin/PromotionCoupon.xml

```diff
@@ -21,8 +21,12 @@
             <operation
                 name="sylius_api_admin_promotion_coupon_get_collection"
                 class="ApiPlatform\Metadata\GetCollection"
-                uriTemplate="/admin/promotion-coupons"
+                uriTemplate="/admin/promotions/{promotionCode}/coupons"
             >
+                <uriVariables>
+                    <uriVariable parameterName="promotionCode" fromClass="Sylius\Component\Core\Model\Promotion"
+                                 fromProperty="coupons"/>
```

Comment:
```suggestion
                    <uriVariable parameterName="promotionCode" fromClass="Sylius\Component\Core\Model\Promotion" fromProperty="coupons"/>
```
or
```suggestion
                    <uriVariable 
                        parameterName="promotionCode" 
                        fromClass="Sylius\Component\Core\Model\Promotion"
                        fromProperty="coupons"
                    />
```

Reply:
I have finally configured the autoformatting properly, so it should be permanently resolved 💃